--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp	2024-10-18 15:18:07.514642435 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:26.612584149 +0200
@@ -947,6 +947,7 @@
 	boolean set;
 
 	set = FALSE;
+	#pragma omp parallel for private(k) lastprivate(set) shared(i,iv,nzv,v,val) 
 	for(k = 0; k < *nzv; k++){
 		if(iv[k] == i){
 			v[k] = val;
