--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.848598627 +0200
@@ -542,6 +542,7 @@
 		 * ---------------------------------------------------------------------
 		 */
 		rho = 0.0;
+		#pragma omp parallel for private(j) shared(alpha,p,q,r,z) 
 		for(j = 0; j < lastcol - firstcol + 1; j++){
 			z[j] = z[j] + alpha*p[j];
 			r[j] = r[j] - alpha*q[j];
