--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp	2024-10-18 15:18:07.514642435 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:26.692546371 +0200
@@ -678,6 +678,7 @@
 		sprnvc(n, nzv, nn1, vc, ivc);
 		vecset(n, vc, ivc, &nzv, iouter+1, 0.5);
 		arow[iouter] = nzv;
+		#pragma omp parallel for private(ivc,ivelt,vc) shared(acol,aelt,iouter,nzv) 
 		for(ivelt = 0; ivelt < nzv; ivelt++){
 			acol[iouter][ivelt] = ivc[ivelt] - 1;
 			aelt[iouter][ivelt] = vc[ivelt];
