--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.976649491 +0200
@@ -754,6 +754,7 @@
 		rowstr[j] = 0;
 	}
 	for(i = 0; i < n; i++){
+		#pragma omp parallel for private(j,nza) shared(acol,arow,i,rowstr) 
 		for(nza = 0; nza < arow[i]; nza++){
 			j = acol[i][nza] + 1;
 			rowstr[j] = rowstr[j] + arow[i];
