--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp	2024-10-18 15:18:07.514642435 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:26.548614372 +0200
@@ -517,6 +517,7 @@
 		 * --------------------------------------------------------------------
 		 */
 		d = 0.0;
+		#pragma omp parallel for private(j) shared(p,q) reduction(+:d) 
 		for (j = 0; j < lastcol - firstcol + 1; j++) {
 			d = d + p[j]*q[j];
 		}
