--- /home/lukas/git/benchmark_harness/explorer_only/burkardt/md/md_orig_with_metadata/md.cpp	2024-10-18 15:16:30.868283820 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/burkardt/md/md_orig_with_metadata/md.cpp.discopop_patch_generator.temp	2024-10-18 15:16:34.188312975 +0200
@@ -307,6 +307,7 @@
 
         pe = pe + 0.5 * pow ( sin ( d2 ), 2 );
 
+        #pragma omp parallel for private(i,rij) shared(d,d2,f,k,nd) 
         for ( i = 0; i < nd; i++ )
         {
           f[i+k*nd] = f[i+k*nd] - rij[i] * sin ( 2.0 * d2 ) / d;
