--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:23.056681326 +0200
@@ -798,6 +798,7 @@
 	size = 1.0;
 	ratio = pow(rcond, (1.0 / (double)(n)));
 	for(i = 0; i < n; i++){
+		#pragma omp parallel for private(goto_40,j,jcol,k,kk,nza,nzrow,scale,va) shared(a,acol,aelt,arow,colidx,i,nzloc,rcond,rowstr,shift,size) 
 		for(nza = 0; nza < arow[i]; nza++){
 			j = acol[i][nza];
 
