--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp	2024-10-18 15:18:07.514642435 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:26.644569038 +0200
@@ -750,6 +750,7 @@
 	 * ...count the number of triples in each row
 	 * --------------------------------------------------------------------
 	 */
+	#pragma omp parallel for private(j) shared(nrows,rowstr) 
 	for(j = 0; j < nrows+1; j++){
 		rowstr[j] = 0;
 	}
