--- /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_without_metadata/IS/is.cpp	2024-10-18 09:31:39.902168415 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_without_metadata/IS/is.cpp.discopop_patch_generator.temp	2024-10-18 09:31:46.674202762 +0200
@@ -380,6 +380,7 @@
 		bucket_size[i] = (INT_TYPE*)alloc_mem(sizeof(INT_TYPE)*NUM_BUCKETS);
 	}
 
+	#pragma omp parallel for private(i,key_buff2) 
 	for( i=0; i<NUM_KEYS; i++ )
 		key_buff2[i] = 0;
 #else /*USE_BUCKETS*/
