--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.792576374 +0200
@@ -471,6 +471,7 @@
 	rho = 0.0;
 
 	/* initialize the CG algorithm */
+	#pragma omp parallel for private(j) shared(p,q,r,x,z) 
 	for(j = 0; j < naa+1; j++){
 		q[j] = 0.0;
 		z[j] = 0.0;
