--- /home/lukas/git/benchmark_harness/explorer_only/burkardt/md/md_orig_with_metadata/md.cpp	2024-10-18 15:16:30.868283820 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/burkardt/md/md_orig_with_metadata/md.cpp.discopop_patch_generator.temp	2024-10-18 15:16:34.160312729 +0200
@@ -367,6 +367,7 @@
   int i;
 
   d = 0.0;
+  #pragma omp parallel for private(i) shared(dr,nd,r1,r2) reduction(+:d) 
   for ( i = 0; i < nd; i++ )
   {
     dr[i] = r1[i] - r2[i];
