--- /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_without_metadata/IS/is.cpp	2024-10-18 09:31:39.902168415 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_without_metadata/IS/is.cpp.discopop_patch_generator.temp	2024-10-18 09:31:46.682202803 +0200
@@ -619,6 +619,7 @@
 		/* Clear the work array section associated with each bucket */
 		k1 = i * num_bucket_keys;
 		k2 = k1 + num_bucket_keys;
+		#pragma omp parallel for private(k) shared(k2,key_buff_ptr) 
 		for ( k = k1; k < k2; k++ )
 			key_buff_ptr[k] = 0;
 		/* Ranking of all keys occurs in this section: */
