--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:23.044676548 +0200
@@ -868,6 +868,7 @@
 		nzloc[j] = nzloc[j] + nzloc[j-1];
 	}
 
+	#pragma omp parallel for private(j,j1,j2,k,nza) shared(a,colidx,nrows,nzloc,rowstr) 
 	for(j = 0; j < nrows; j++){
 		if(j > 0){
 			j1 = rowstr[j] - nzloc[j-1];
