--- /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_with_metadata/IS/is.cpp	2024-10-18 09:31:39.886168334 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_with_metadata/IS/is.cpp.discopop_patch_generator.temp	2024-10-18 09:31:43.666187513 +0200
@@ -434,6 +434,7 @@
 
 	k = MAX_KEY/4;
 
+	#pragma omp parallel for private(i,key_array,x) shared(an,k,k2) 
 	for(i=k1; i<k2; i++){
 		x = randlc(&s, an);
 		x += randlc(&s, an);
