--- /home/lukas/git/benchmark_harness/explorer_only/burkardt/md/md_orig_with_metadata/md.cpp	2024-10-18 15:16:30.868283820 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/burkardt/md/md_orig_with_metadata/md.cpp.discopop_patch_generator.temp	2024-10-18 15:16:34.172312835 +0200
@@ -278,6 +278,7 @@
   
 
 //# pragma omp for reduction ( + : pe, ke )
+  #pragma omp parallel for private(d,d2,i,j,k,rij) shared(PI2,f,nd,np,pos,vel) reduction(+:ke,pe) 
   for ( k = 0; k < np; k++ )
   {
 //
