--- /home/lukas/git/benchmark_harness/explorer_only/burkardt/md/md_orig_with_metadata/md.cpp	2024-10-18 15:16:30.868283820 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/burkardt/md/md_orig_with_metadata/md.cpp.discopop_patch_generator.temp	2024-10-18 15:16:34.152312659 +0200
@@ -316,6 +316,7 @@
 //
 //  Compute the kinetic energy.
 //
+    #pragma omp parallel for private(i) shared(k,nd,vel) reduction(+:ke) 
     for ( i = 0; i < nd; i++ )
     {
       ke = ke + vel[i+k*nd] * vel[i+k*nd];
