--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp	2024-10-18 15:18:07.514642435 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:26.624578482 +0200
@@ -923,6 +923,7 @@
 		 * --------------------------------------------------------------------
 		 */
 		boolean was_gen = FALSE;
+		#pragma omp parallel for private(ii) shared(i,iv,nzv) 
 		for(ii = 0; ii < nzv; ii++){
 			if(iv[ii] == i){
 				was_gen = TRUE;
