--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.860603396 +0200
@@ -260,6 +260,7 @@
 	 * to local, i.e., (0 --> lastcol-firstcol)
 	 * ---------------------------------------------------------------------
 	 */
+	#pragma omp parallel for private(j,k) 
 	for(j = 0; j < lastrow - firstrow + 1; j++){
 		for(k = rowstr[j]; k < rowstr[j+1]; k++){
 			colidx[k] = colidx[k] - firstcol;
