--- /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_without_metadata/IS/is.cpp	2024-10-18 09:31:39.902168415 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_without_metadata/IS/is.cpp.discopop_patch_generator.temp	2024-10-18 09:31:46.650202641 +0200
@@ -529,6 +529,7 @@
 
 	/* Confirm keys correctly sorted: count incorrectly sorted keys, if any */
 	j = 0;
+	#pragma omp parallel for private(i) shared(key_array) reduction(+:j) 
 	for( i=1; i<NUM_KEYS; i++ )
 		if( key_array[i-1] > key_array[i] )
 			j++;
