--- /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_without_metadata/IS/is.cpp	2024-10-18 09:31:39.902168415 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_without_metadata/IS/is.cpp.discopop_patch_generator.temp	2024-10-18 09:31:46.662202702 +0200
@@ -592,6 +592,7 @@
 
 	for( i=1; i< NUM_BUCKETS; i++ ) { 
 		bucket_ptrs[i] = bucket_ptrs[i-1];
+		#pragma omp parallel for private(bucket_ptrs,bucket_size,k) shared(i,myid) 
 		for( k=0; k< myid; k++ )
 			bucket_ptrs[i] += bucket_size[k][i];
 		for( k=myid; k< num_procs; k++ )
