--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp	2024-10-18 15:18:07.514642435 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:26.636572816 +0200
@@ -267,6 +267,7 @@
 	}
 
 	/* set starting vector to (1, 1, .... 1) */
+	#pragma omp parallel for private(i) 
 	for(i = 0; i < NA+1; i++){
 		x[i] = 1.0;
 	}
