--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:23.068686105 +0200
@@ -284,6 +284,7 @@
 	 * do one iteration untimed to init all code and data page tables
 	 * ----> (then reinit, start timing, to niter its)
 	 * -------------------------------------------------------------------*/
+	#pragma omp parallel for private(it,norm_temp1,norm_temp2) 
 	for(it = 1; it <= 1; it++){
 		/* the call to the conjugate gradient routine */
 		conj_grad(colidx, rowstr, x, z, a, p, q, r, &rnorm);
