--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.688535047 +0200
@@ -584,6 +584,7 @@
 	sum = 0.0;
 	for(j = 0; j < lastrow - firstrow + 1; j++){
 		d = 0.0;
+		#pragma omp parallel for private(k) shared(a,colidx,j,rowstr,z) reduction(+:d) 
 		for(k = rowstr[j]; k < rowstr[j+1]; k++){
 			d = d + a[k]*z[colidx[k]];
 		}
