--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.956641544 +0200
@@ -355,6 +355,7 @@
 		printf("    %5d       %20.14e%20.13e\n", it, rnorm, zeta);
 
 		/* normalize z to obtain x */
+		#pragma omp parallel for private(j) shared(norm_temp2) 
 		for(j = 0; j < lastcol - firstcol + 1; j++){
 			x[j] = norm_temp2 * z[j];
 		}
