--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.988654260 +0200
@@ -882,6 +882,7 @@
 			nza = nza + 1;
 		}
 	}
+	#pragma omp parallel for private(j) shared(nrows,nzloc,rowstr) 
 	for(j = 1; j < nrows+1; j++){
 		rowstr[j] = rowstr[j] - nzloc[j-1];
 	}
