--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.944636776 +0200
@@ -817,6 +817,7 @@
 				}
 
 				goto_40 = FALSE;
+				#pragma omp parallel for lastprivate(k) shared(colidx,j,jcol,rowstr) 
 				for(k = rowstr[j]; k < rowstr[j+1]; k++){
 					if(colidx[k] > jcol){
 						/*
