--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.932632007 +0200
@@ -824,6 +824,7 @@
 						 * ... insert colidx here orderly
 						 * ----------------------------------------------------------------
 						 */
+						#pragma omp parallel for private(kk) shared(a,colidx,k) 
 						for(kk = rowstr[j+1]-2; kk >= k; kk--){
 							if(colidx[kk] > -1){
 								a[kk+1] = a[kk];
