--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp	2024-10-18 15:18:07.514642435 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:26.600589816 +0200
@@ -345,6 +345,7 @@
 		 */
 		norm_temp1 = 0.0;
 		norm_temp2 = 0.0;
+		#pragma omp parallel for private(j) reduction(+:norm_temp1,norm_temp2) 
 		for(j = 0; j < lastcol - firstcol + 1; j++){
 			norm_temp1 = norm_temp1 + x[j]*z[j];
 			norm_temp2 = norm_temp2 + z[j]*z[j];
