--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:23.020666991 +0200
@@ -782,6 +782,7 @@
 	 * ... preload data pages
 	 * ---------------------------------------------------------------------
 	 */
+	#pragma omp parallel for private(j,k) shared(a,colidx,nrows,nzloc,rowstr) 
 	for(j = 0; j < nrows; j++){
 		for(k = rowstr[j]; k < rowstr[j+1]; k++){
 			a[k] = 0.0;
