--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.676530278 +0200
@@ -595,6 +595,7 @@
 	 * at this point, r contains A.z
 	 * ---------------------------------------------------------------------
 	 */
+	#pragma omp parallel for private(d,j) shared(r,x) reduction(+:sum) 
 	for(j = 0; j < lastcol-firstcol+1; j++){
 		d   = x[j] - r[j];
 		sum = sum + d*d;
