--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:23.000659028 +0200
@@ -876,6 +876,7 @@
 		}
 		j2 = rowstr[j+1] - nzloc[j];
 		nza = rowstr[j];
+		#pragma omp parallel for private(k,nza) shared(a,colidx,j2) 
 		for(k = j1; k < j2; k++){
 			a[k] = a[nza];
 			colidx[k] = colidx[nza];
