--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.816585911 +0200
@@ -270,6 +270,7 @@
 	for(i = 0; i < NA+1; i++){
 		x[i] = 1.0;
 	}
+	#pragma omp parallel for private(j) 
 	for(j = 0; j<lastcol-firstcol+1; j++){
 		q[j] = 0.0;
 		z[j] = 0.0;
