--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:23.012663806 +0200
@@ -503,6 +503,7 @@
 		 * the unrolled-by-8 version below is significantly faster
 		 * on the Cray t3d - overall speed of code is 1.5 times faster.
 		 */
+		#pragma omp parallel for private(j,k,sum) shared(a,colidx,p,q,rowstr) 
 		for(j = 0; j < lastrow - firstrow + 1; j++){
 			sum = 0.0;
 			for(k = rowstr[j]; k < rowstr[j+1]; k++){
