--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp	2024-10-18 15:18:07.514642435 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_without_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:26.656563371 +0200
@@ -305,6 +305,7 @@
 		norm_temp2 = 1.0 / sqrt(norm_temp2);
 
 		/* normalize z to obtain x */
+		#pragma omp parallel for private(j) shared(norm_temp2) 
 		for(j = 0; j < lastcol - firstcol + 1; j++){     
 			x[j] = norm_temp2 * z[j];
 		}
