--- /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp	2024-10-18 15:18:07.506639426 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/CG/CG_orig_with_metadata/CG/cg.cpp.discopop_patch_generator.temp	2024-10-18 15:18:22.784573195 +0200
@@ -261,6 +261,7 @@
 	 * ---------------------------------------------------------------------
 	 */
 	for(j = 0; j < lastrow - firstrow + 1; j++){
+		#pragma omp parallel for private(k) shared(j) 
 		for(k = rowstr[j]; k < rowstr[j+1]; k++){
 			colidx[k] = colidx[k] - firstcol;
 		}
