--- /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_with_metadata/IS/is.cpp	2024-10-18 09:31:39.886168334 +0200
+++ /home/lukas/git/benchmark_harness/explorer_only/NPB/IS/IS_orig_with_metadata/IS/is.cpp.discopop_patch_generator.temp	2024-10-18 09:31:43.686187613 +0200
@@ -594,6 +594,7 @@
 		bucket_ptrs[i] = bucket_ptrs[i-1];
 		for( k=0; k< myid; k++ )
 			bucket_ptrs[i] += bucket_size[k][i];
+		#pragma omp parallel for private(k) shared(bucket_ptrs,bucket_size,i,num_procs) 
 		for( k=myid; k< num_procs; k++ )
 			bucket_ptrs[i] += bucket_size[k][i-1];
 	}
